packages/tables/src/Columns/Concerns/HasColors.php

```diff
@@ -23,7 +23,7 @@ public function getStateColor(): ?string
         foreach ($this->getColors() as $color => $condition) {
             if (is_numeric($color)) {
                 $stateColor = $condition;
-            } elseif ($condition instanceof Closure && $condition($state)) {
+            } elseif ($condition instanceof Closure && $condition($state, $this->getRecord())) {
```

Comment:
Shouldn't we use `$this->evaluate()` here instead?

```suggestion
            } elseif ($condition instanceof Closure && $this->evaluate($condition, ['state' => $state])) {
```

This could break stuff for people relying on argument position instead of name though.

Reply:
Yeah, this would be a breaking change unfortunately.

Reply:
Really? If people are using `$state` as documented, it shouldn't be

Reply:
Probably depends on what you see as breaking change. When people adhere strictly to the docs, this shouldn't be an issue. I'd prefer to break it now instead of waiting longer.

Reply:
I'd still avoid it, I know I have at least 1 app that doesn't call it $state.

Reply:
Shame. I'll add it to a breaking changes list for v3.